### PR TITLE
NMS-14692: Allow destination paths to be triggered from the shell and UI

### PR DESCRIPTION
**Pull request by j-white**
JIRA: https://opennms.atlassian.net/browse/NMS-14692

There's a button now!

image

### CODE REVIEW

**Review by christianpape — CHANGES_REQUESTED** [review]
Really useful. Code-wise fine for me. It would be nice to have some of this described in the documentation, especially Karaf command and the ReST endpoint.

**Review by aramosvizcarra — DISMISSED** [review]
LGTM

**Comment by Bonrob2** [comment]
> Really useful. Code-wise fine for me. It would be nice to have some of this described in the documentation, especially Karaf command and the ReST endpoint.

There is some documentation on this, but not the Karaf/REST endpoint: https://docs.opennms.com/horizon/31/operation/quick-start/notifications.html#configure-destination-paths

**Review by Bonrob2 — COMMENTED** [review]
text LGTM!

**Comment by j-white** [comment]
Added some basic reference documentation.

File: docs/modules/operation/pages/deep-dive/notifications/shell.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
+
+= Notification Shell Commands
+
+The following shell commands are made available to help administer and debug Notifd.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/notifications/shell.adoc:4]:
```suggestion
The following shell commands can help you to administer and debug Notifd.
```

File: docs/modules/operation/pages/deep-dive/notifications/shell.adoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+== opennms:destination-path-trigger
+
+Use the `destination-path-trigger` command to test the notification targets and commands associated with a destination path.
+This can be used to verify that commands are properly configured and that notifications get sent to the recipients.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/notifications/shell.adoc:9]:
```suggestion
Use to verify that commands are properly configured and that notifications get sent to the recipients.
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
A few minor changes to the docs.

**Comment by j-white** [comment]
Should be all set now :)

**Review by Bonrob2 — APPROVED** [review]
docs LGTM!

**Review by christianpape — APPROVED** [review]
LGTM.

**Comment by RangerRick** [comment]
I'l go ahead and merge this since @j-white is out until next week

**Pull request merged**